site/content/vendir/docs/develop/sync.md

```diff
@@ -39,3 +39,37 @@ To use these resolved references on top of `vendir.yml`, use `vendir sync -l`.
 ## Syncing from different directory
 
 As of v0.22.0, you can use `--chdir` flag with `vendir sync` command to change current working directory of vendir before any syncing occurs. All other paths provided to `vendir sync` should be relative to the changed directory.
+
+## Caching
+
+`vendir` allows the users to cache OCI Images in their disk so as not to rely on access to the registry all the time.
+To activate this feature the users need to set the environment variable `VENDIR_CACHE_DIR`. This variable should point
+to the path where they want to store the OCI Images.
+
+```bash-plain
+$ mkdir -p ~/.vendir/cache
+$ export VENDIR_CACHE_DIR=~/.vendir/cache
+$ vendir sync
+```
+
+**Note:** Not all images can be cached. Only images that are referenced by SHA can be cached.
+
+The user can also specify what is the maximum size of the content to be cached, by setting the environment variable
+`VENDIR_MAX_CACHE_SIZE`. The default value, when the variable is not provided is `1Mi` (1 Megabyte).
```

Comment:
i realize this is just the docs, but is that default size too small to ever be useful ?   i wouldn't have been surprised if it were a G instead of an M,  but should we at least put a zero or two after that 1 ?

Reply:
Good question. The base usecase that we currently have for this feature is the usage of vendir by kapp-controller. In this case kapp-controller will download images or bundles but they are yaml configurations almost exclusively, which means that the images to be cached are going to be quite small, in the majority of the cases sub 5kb.